### PR TITLE
Ease startup

### PR DESCRIPTION
**Pull request by themouette**
Add some utilities to fasten develoment startup.
- [X] Add install script
- [X] Add Gemfile
- [X] Add fig file
- [X] Add npm scripts
- [x] Update documentation

This allows to have a local mongodb instance without having to install it

### CODE REVIEW

**Comment by anselmbradford** [comment]
Appears fig has been deprecated in favor Docker Compose http://www.fig.sh/yml.html

File: Gruntfile.js [hunk 1]
```diff
@@ -80,17 +80,21 @@ module.exports = function(grunt) {
...
+//            express: {
+//                files: [
+//                    'server.js',
+//                    'server/**/*.{js,json}'
+//                ],
+//                tasks: ['newer:jshint:server', 'express:dev', 'wait'],
+//                options: {
+//                    livereload: true,
+//                    nospawn: true //Without this option specified express won't be reloaded
+//                }
```
Comment (christian-bromann) [inline Gruntfile.js]: if this isn't required anymore we can delete it

**Comment by christian-bromann** [comment]
@themouette I would like to merge this. Could you fix the issues above?

**Comment by GeorgeTaveras1231** [comment]
I opened #31 that focuses on introducing docker into this project. I realized there were still a lot of changes that needed to be made in this PR besides the things that @christian-bromann commented on.

For example: Fig has been deprecated; fig.yml file has a mongo db container.

@christian-bromann Please take a look at #31 when you get the chance. These changes will make it much easier for me to introduce webdrivercss to my team.

**Pull request closed without merging**